plugin.json

```diff
@@ -15,18 +15,17 @@
         }
     },
     "settings_schema": {
-        "header": "Antivirus plugin for scanning uploaded files to Mattermost, uses ClamAV to scan files. See [documentation here](https://github.com/mattermost/mattermost-plugin-antivirus)",
-        "footer": "",
+        "header": "Antivirus plugin for scanning uploaded files to Mattermost, uses ClamAV to scan files. See [documentation here](https://github.com/mattermost/mattermost-plugin-antivirus)."
```

Comment:
```suggestion
        "header": "Antivirus plugin which uses ClamAV to scan files uploaded to Mattermost. See [documentation here](https://github.com/mattermost/mattermost-plugin-antivirus)."
```

Reply:
@iomodo, I reworded it a bit. LMK what you think?

Reply:
@justinegeffen Wording is great, just missing the trailing comma for the json to be valid

Reply:
Oh my gosh, again. Sorry!